frontend/src/metabase/containers/SaveQuestionModal/SaveQuestionModal.tsx

```diff
@@ -27,7 +25,7 @@ export const SaveQuestionModal = ({
     multiStep={multiStep}
     initialCollectionId={initialCollectionId}
   >
-    <Modal.Root onClose={onCancel} opened={true} padding="40px">
+    <Modal.Root padding="2.5rem" {...modalProps}>
```

Comment:
I've moved the logic for the modal props to each iteration of `SaveQuestionModal`. In an ideal world I hope we can use the modal in a proper way rather than having to use `opened=true` for all of it